packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -39,57 +46,51 @@ export const getSplitButtonVerticalDividerClassNames = memoizeFunction(
   }
 );
 
-export const getContextualMenuClassNames = memoizeFunction(
-  (theme: ITheme, className?: string): IContextualMenuClassNames => {
-    const styles = getContextualMenuStyles(theme);
-
-    return mergeStyleSets({
-      container: [
-        'ms-ContextualMenu-container',
-        styles.container,
-        className,
-        [
-          {
-            selectors: {
-              ':focus': { outline: 0 }
-            }
-          }
-        ]
-      ],
-      root: ['ms-ContextualMenu is-open', styles.root],
-      list: ['ms-ContextualMenu-list is-open', styles.list],
-      header: ['ms-ContextualMenu-header', styles.header],
-      title: styles.title
-    });
-  }
-);
+const GlobalClassNames = {
+  item: 'ms-ContextualMenu-item',
+  divider: 'ms-ContextualMenu-divider',
+  root: 'ms-ContextualMenu-link',
+  isChecked: 'is-checked',
+  isExpanded: 'is-expanded',
+  isDisabled: 'is-disabled',
+  linkContent: 'ms-ContextualMenu-linkContent',
+  linkContentMenu: 'ms-ContextualMenu-linkContent',
+  icon: 'ms-ContextualMenu-icon',
+  checkmarkIcon: 'ms-ContextualMenu-checkmarkIcon',
+  subMenuIcon: 'ms-ContextualMenu-submenuIcon',
+  label: 'ms-ContextualMenu-itemText',
+  secondaryText: 'ms-ContextualMenu-secondaryText'
+};
 
 export const getItemClassNames = memoizeFunction(
```

Comment:
>memoizeFunction [](start = 33, length = 15)

I think @dzearing had a specific scenario where memoizing the function causes undesirable side effects. I don't know the details of it but this is why the classNamesFunction method itself does not memoize (though that layer of indirection was precisely to add global memoization initially).

Reply:
I'm open to addressing this just not sure what to look for. I kept it as-is to avoid changing more than necessary given what else I had to do throughout here.